apollo-compiler/src/test/graphql/com/example/arguments_complex/TestQuery.kt

```diff
@@ -170,21 +171,34 @@ data class TestQuery(
           ResponseField.forObject("heroWithReview", "heroWithReview", mapOf<String, Any>(
             "episode" to mapOf<String, Any>(
               "kind" to "Variable",
-              "variableName" to "episode"),
+              "variableName" to "episode"
+            ),
             "review" to mapOf<String, Any>(
               "stars" to mapOf<String, Any>(
                 "kind" to "Variable",
-                "variableName" to "stars"),
+                "variableName" to "stars"
+              ),
               "favoriteColor" to mapOf<String, Any>(
-                "red" to "0",
+                "red" to 0,
```

Comment:
I also took this opportunity to map GraphQL's Int and Float to Kotlin's Int and Double for better typing. While I believe it's more correct, it's a behaviour change so I'm happy to revert if we think this is risky.

Reply:
Thinking more about this, this shouldn't change what's sent to the server (which is in `QUERY_DOCUMENT`) so the impact would only be for the cache. So impact is still limited but that could trigger some unwanted cache miss so I'm going to revert that.